### PR TITLE
CoinStats Webapp metrics M3 capstone

### PR DESCRIPTION
**Pull request by lerfast**
Hello fellow code reviewer, I did the following task for this Capstone project:

- [ ] I Chose the Coinstat API to work with: https://api.coinstats.app/public/v1/coins
- [ ] 
- [ ] I choose the design color form  [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), including:

 Colors (select one main color for your website).

 Typography: font face, size and weight.

 Layout: composition and space between elements.

Interactions:

Home page

- [ ] When the page loads, the web app shows the list of categories that could be filtered out by some parameter, I user can filtered by crypto name

- [ ]  It displays some numeric values that come from the API.

- [ ]  When the user clicks (or taps) on a category item, the application navigates to the details page.

Details page

- [ ] In the details page, the web app retrieves data from the API to show detailed data in the category.

- [ ]  When the user clicks on the "Back" button (<), the user navigates to the home page.

- [ ] Write the test files for componentes HomePage.js, DetailsPage.js and NavBar.js

- [ ] The Loom video I will post it when I get the approval and everything is okay

- [ ] Live demo of this project [here](https://cryptocoins-stats.onrender.com/))

### CODE REVIEW

**Comment by lerfast** [comment]
image

File: README.md [hunk 1]
```diff
@@ -1,2 +1,197 @@
...
+- **Following best practices for HTML, CSS, JS**
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+🚀 Live demo of this project [here](https://cryptocoins-stats.onrender.com/)
+
+🚀 Video Info of this project [here](Comming soon)
```
Comment (julie-ify) [inline README.md]: - [ ] From your PR summary you mentioned you will add the video after the project is approved, but video presentation is part of the requirements for this project to be approved so you still need to record the video before approaval.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @lerfast,

Good job so far!

## Highlights!
- Unique coin routes ✔️ 
- Checkered color pattern
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (lerfast) [reply README.md]: Hi Julie, thank you!!!, 
Done!!!
image

You can check [here](https://www.loom.com/share/25395352af5441a890adfc54ffc13f7c)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,197 @@
...
+- **Following best practices for HTML, CSS, JS**
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+🚀 Live demo of this project [here](https://cryptocoins-stats.onrender.com/)
+
+🚀 Video Info of this project [here](https://www.loom.com/share/25395352af5441a890adfc54ffc13f7c)
```
Comment (danmainah) [inline README.md]: - [ ] @lerfast Great work adding the video presentation, however, you forgot to highlight a piece of the code that you find interesting or proud of making. Kindly consider ensuring your presentation fulfills all the project requirements shown below
image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @lerfast , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️  Implemented the project requirement ensuring all pages are working correctly and the pages have a great design following the given design.
✔️  Ensured the app has tests and all tests are passing.


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (lerfast) [reply README.md]: image
I recorded again the video, and I am sorry I fell nervous in front of a camera, I have to work with that, and I hope that this time it makes it clearer.

**Review by DammyShittu — APPROVED** [review]
Hi @lerfast,

## APPROVED 🟢

## Highlights
✔️ Detailed video presentation

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**